Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+  } 
+
+  stages {
+    stage('Test') {
+      steps {
+        sh './test.sh'
```

Comment:
We can split up `test.sh` and make it into another issue for another PR. That will turn this into a real pipeline with steps. Really helps with discovery and debugging, it turns out.